custom_components/frigate/media_source.py

```diff
@@ -395,20 +393,15 @@ def _validate_hour(
 class RecordingIdentifier(Identifier):
     """Recording Identifier."""
 
-    year_month: str | None = attr.ib(
-        default=None,
-        validator=[
-            attr.validators.instance_of((str, type(None))),
-            _validate_year_month,
-        ],
+    camera: str | None = attr.ib(
```

Comment:
Caution: This changes the media content id for recordings for Frigate, so will break anything that uses these identifiers to fetch recordings. This is at least (but probably only) the card.

Reply:
[This is not a problem with this PR, but I'll have to get a new card build out at some point to fix the recordings]